in_toto/model_test.go

```diff
@@ -265,13 +266,13 @@ func TestMetablockVerifySignature(t *testing.T) {
 	}
 	expectedErrors := []string{
 		"No signature found",
-		"verification error",
+		"encoding/hex: invalid byte: U+0020 ' '",
 		"json: unsupported type",
 	}
 	for i := 0; i < len(mbs); i++ {
 		err := mbs[i].VerifySignature(key)
 		if err == nil || !strings.Contains(err.Error(), expectedErrors[i]) {
-			t.Errorf("Metablock.VerifySignature returned '%s', expected '%s'",
+			t.Errorf("Metablock.VerifyRSASignature returned '%s', expected '%s'",
```

Comment:
```suggestion
			t.Errorf("Metablock.VerifySignature returned '%s', expected '%s'",
```

Reply:
Fixed in https://github.com/in-toto/in-toto-golang/pull/56/commits/236acb803e2318f44e0c7c32c717e63729c0dd42